modules/unshorten/index.js

```diff
@@ -0,0 +1,30 @@
+const request = require('request');
+
+const use = {
+  unshorten: 'unshorten url shortenings',
+};
+
+function unshorten(link, reply) {
+  request(link, (error, response) => {
+    if (!error && response.statusCode === 200) {
+      reply(`${response.request.uri.href}`);
+    } else {
+      reply(`${link}: redirect fails`);
+    }
+  });
+}
+
+
+function usage(command, reply) {
+  reply(`${command} ${use[command]}`);
```

Comment:
This should be one reply with a newline in the text; In this case it's no different, but in general reply has internal logic to handle multi-line content, and well behaved modules shouldn't avoid that without good reason.

Reply:
They should be joined now.